.gitignore

```diff
@@ -24,4 +24,8 @@ target
 # mac
 .DS_Store
 
-.idea
\ No newline at end of file
+.idea
+.classpath
+.project
+.settings
+
```

Comment:
Thanks for adding those :)
Could you also add .vscode? Its a directory created by vscode editor to keep workspace settings etc.